.github/workflows/clients.yml

```diff
@@ -26,7 +26,15 @@ jobs:
     - uses: ./.github/actions/setup-contracts-clients
     - run: yarn build
 
-  test:
+  test-unit:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./.github/actions/setup-contracts-clients
+    - run: yarn test
+
+  test-integration:
```

Comment:
@JohnGuilding I split this into separate steps to make it easier to identify unit vs. integration test failures. It will also make runs slightly faster, especially if we add more unit tests.